client/state/data-layer/README.md

```diff
@@ -106,6 +105,10 @@ Let's look at a full example:
  *
  * Requests splines and reticulations from API
  * and feeds the responses back into Calypso
+ *
+ * Note: This is an example, but we should be using
+ *       the HTTP action types instead of `wpcom.js`
+ * @see ./wpcom-http/README.md
```

Comment:
I'm thinking we should maybe update the example here to be clear. I'm not sure how many folks would notice this while skimming.